geomesa-compute/src/main/scala/org/locationtech/geomesa/compute/spark/GeoMesaSpark.scala

```diff
@@ -186,27 +192,20 @@ class GeoMesaSparkKryoRegistrator extends KryoRegistrator {
 
   override def registerClasses(kryo: Kryo): Unit = {
     val serializer = new com.esotericsoftware.kryo.Serializer[SimpleFeature]() {
-      val typeCache = CacheBuilder.newBuilder().build(
-        new CacheLoader[String, SimpleFeatureType] {
-          override def load(key: String): SimpleFeatureType = {
-            val spec = System.getProperty(GeoMesaSpark.typeProp(key))
```

Comment:
i think we still need this to pass sfts between the jvms in a spark cluster. the static hashmap below will only work inside a single jvm. the key piece is that the system props get passed around the cluster through the 'extraJavaOptions' config.